components/SponsorsSection.js

```diff
@@ -57,7 +57,7 @@ export default function TicketsSection () {
 function ItemSponsor ({ src, type }) {
   return (
     <div className={`sponsor-item-${type}`}>
-      <img src={src} width='100%' />
+      <img alt='Photo' src={src} width='100%' />
```

Comment:
(Note that, for the Speakers section, `alt='Photo'` is used because the text immediately following that image is the name of the speaker.)